examples/python/basic/build-xml.py

```diff
@@ -15,21 +15,16 @@
 #                 Exporting to OpenMC materials.xml file
 ###############################################################################
 
-# Instantiate some Nuclides
-h1 = openmc.Nuclide('H1')
-o16 = openmc.Nuclide('O16')
-u235 = openmc.Nuclide('U235')
-
 # Instantiate some Materials and register the appropriate Nuclides
 moderator = openmc.Material(material_id=41, name='moderator')
 moderator.set_density('g/cc', 1.0)
-moderator.add_nuclide(h1, 2.)
-moderator.add_nuclide(o16, 1.)
+moderator.add_element('H', 2.)
```

Comment:
It's always been a curiosity of mine which is more wrong: Putting only H1 in water? Or putting both H1 and H2 in water but only using a S(a,b) table for H1 (effectively what this does)?